merge-process.md

```diff
@@ -1,27 +1,21 @@
 # Merge process
 
 When a pull request (PR) is issued, all interested parties should express any
-concerns they have about the PR, by adding comments to the PR. The commit team
-(members wth write-access to the repository) will generally wait for at least
-one full business day (Mon - Fri) before beginning voting to accept the PR, to
-provide time for people to comment.
+concerns they have about the PR, by adding comments to the PR.
 
 Acceptance of a PR is accomplished as follows:
 
-- When the stream of comments has died down -- in particular, dissenting
-  comments -- a member of the commit team will request reviews from the other
-  team members.
 - If the PR proposer is a member of the commit team, he/she does not have voting
   rights on this PR.
-- A PR will be accepted when one of the following conditions has been met:
+- A PR will be accepted when at least three business days have passed since the call 
+  for votes, and one of the following conditions has been met:
```

Comment:
I think we should use "since the reviews started" instead of "since the call for votes"